js/private/js_expand_template.js

```diff
@@ -0,0 +1,87 @@
+const fs = require('fs')
+
+/**
+ * The status files are expected to look like
+ * BUILD_SCM_HASH 83c699db39cfd74526cdf9bebb75aa6f122908bb
+ * BUILD_SCM_LOCAL_CHANGES true
+ * STABLE_BUILD_SCM_VERSION 6.0.0-beta.6+12.sha-83c699d.with-local-changes
+ * BUILD_TIMESTAMP 1520021990506
+ *
+ * Parsing regex is created based on Bazel documentation describing the status file schema:
+ *   The key names can be anything but they may only use upper case letters and underscores. The
+ *   first space after the key name separates it from the value. The value is the rest of the line
+ *   (including additional whitespace).
+ */
+function _parseStatusFile(p) {
+    if (!p) return []
+    const results = {}
+    const statusFile = fs.readFileSync(p, { encoding: 'utf-8' })
+    for (const match of `\n${statusFile}`.matchAll(/^([A-Z_]+) (.*)/gm)) {
```

Comment:
Is it a requirement that it's all caps?

Can the name/value be separated by more then one space? I think `\s+` would work?

Reply:
Don't think it is a requirement. Nice catch.

Reply:
Fixed ✅